packages/lib-user/README.md

```diff
@@ -36,6 +36,7 @@ Note: query params are used for local development work, but are not used in prod
 - `https://www.zooniverse.org/users/[login]/stats`
 - `https://www.zooniverse.org/users/[login]/stats/certificate`
 - `https://www.zooniverse.org/groups/[user group ID]`
+- `https://local.zooniverse.org:8080/?groups=[groupId]`
```

Comment:
This is duplicative of line 32? I think this list is intended to be (eventual) production urls and the lines before (27-32) note development urls.

Reply:
Yes you're right! I totally missed that bit.